friendly_traceback/info_generic.py

```diff
@@ -40,7 +46,7 @@ def wrapper():
 
 
 @register(ArithmeticError)
-def arithmetic_error():
+def arithmetic_error() -> str:
```

Comment:
@aroberge those functions are not exposed, so the type hints are not really necessary (were added automatically by my type hints generator scripts). Say the word and I will remove them.

Reply:
It should not hurt to keep them and it may turned out to be useful. Furthermore, you fixed an issue via https://github.com/friendly-traceback/friendly-traceback/pull/5 which would have been caught if this type of type hint would have been added.